### ISSUE TITLE
Error after clicking title screen

### ISSUE THREAD

**Issue by Akhmadi14** [Xele02/UMO#130]
Got the data dump from my smartphone and decrypt it. 
Run UMO and title screen appear, after clicking it got black screen, cursor and BGM music are still working.

error log:
TODO
UnityEngine.Debug:LogError(Object)
TodoLogger:LogError(Int32, String) (at Assets/Scripts/TodoLogger.cs:75)
<LGFPCADOCAA_ShowError>d__33:MoveNext() (at Assets/UMAssets/Scripts/IMMAOANGPNK.cs:337)
UnityEngine.MonoBehaviour:StartCoroutine(IEnumerator)
MonoBehaviourExtensions:StartCoroutineWatched(MonoBehaviour, IEnumerator) (at Assets/Scripts/CoroutineWatcher.cs:13)
<MHEKMICKGDM_LoadFromStorage>d__32:MoveNext() (at Assets/UMAssets

**Comment by Xele02** [comment]
Hello,
From what I unterstand, yuo try to run the game on PC. Where did you get the data, when you say from your phone, is it from the old game ?

**Comment by Akhmadi14** [comment]
Its from the game just before the server shutdown (around June 2022 I think).
I copied the android and db folders from my smartphone to my PC just as the instruction says (took almost 1 full day to copy all the files).

However I noticed that the directory structure from my game is rather different from the one you posted on the wiki, noticeably is the ar folder.
Is there something wrong with my game data?
It works fine in my smartphone before the server shutdown.

**Comment by Xele02** [comment]
The original game was downloading data on demand and deleting unused one to not take too much place on the phone. But you need a full dump now for it to works. This can either be done :
- With the tools here if the server still works : https://github.com/Xele02/UMO/tree/main/Tools
- By downloading one of the dump that was done. I'll see if we still have something uploaded ready somewhere.

**Comment by Xele02** [comment]
You can dld the dump on this google drive. Tell me when you got the link, I'll delete this message : https://drive.google.com/file/d/16RsoJ7uOMcwp_sNQijNwz8DXAyVJpryD/view?usp=drivesdk

**Comment by Akhmadi14** [comment]
Ok, I got the link.
You can delete the link now.

I'll update as soon as I finished with the data dump.

**Comment by Akhmadi14** [comment]
Did a clean install of UMO 5.0 and use th dl game data.
Still stuck on blackscreen after title.
These are the error logs:

TODO
UnityEngine.Debug:LogError(Object)
TodoLogger:LogError(Int32, String) (at Assets/Scripts/TodoLogger.cs:76)
JHHBAFKMBDL:GKMAHMLNMEK(IMCBBOAFION, String) (at Assets/UMAssets/Scripts/JHHBAFKMBDL.cs:239)
<MHEKMICKGDM_LoadFromStorage>d__33:MoveNext() (at Assets/UMAssets/Scripts/IMMAOANGPNK.cs:328)
UnityEngine.SetupCoroutine:InvokeMoveNext(IEnumerator, IntPtr) (at C:/buildslave/unity/build/Runtime/Export/Coroutines.cs:17)

Something wrong with my Unity setup?

**Comment by Xele02** [comment]
Yeah, but I don't think it's from the data. Can you link the full log which can be found here :
Operating system | Log files
-- | --
Linux | ~/.config/unity3d/Editor.log
macOS | ~/Library/Logs/Unity/Editor.log
Windows | C:\Users\username\AppData\Local\Unity\Editor\Editor.log

**Comment by Akhmadi14** [comment]
https://github.com/Akhmadi14/Others/blob/main/Editor.log

**Comment by Xele02** [comment]
Thx, I'll try to see if I understand what's wrong. In the mean time, you can try  to delete the directory C:/Users/username/AppData/LocalLow/UtaMacross/UtaMacross/ . This is where are saved some local file and perhaps the first run with your phone data wrote something that the current version don't like.

**Comment by Xele02** [comment]
And to be sure, the data of the full dump is the one in "F:\UMO\Tools\Data/" (this is where the editor search in the log) , and you removed the previous one if it was in the same directory.

**Comment by Xele02** [comment]
Ok, I'll try to do test on fresh account to see if I have the problem cause for now I don't really why this is happening

**Comment by Xele02** [comment]
Hello, I try to do some test, but didn't had your problem :
- Removed the profile directory
- Dld and extract 0.5 zip
- Dld the data from the google drive and extracted.
- Opened unity hub and loaded theproject in the Unity Directory.
- In the editor, Menu UMO > Start
- Selected the directory with the data from the gg drive.
- Game started normally.

So for another test, can you open the option (UMO / Options) and change the log number to 999 :
![image](https://github.com/Xele02/UMO/assets/2817631/00e3c75b-5c86-4af3-8d27-927cb50cc312)
And resend the unity log after running again the game with those options. I will see if I can find more info with more log.

**Comment by Akhmadi14** [comment]
This are the newest editor log.
https://github.com/Akhmadi14/Others/blob/main/Editor.log
I did the same step as you above, and still doesn't work. I suspect something in my Unity asset might be corrupt or something.
I'm using Unity editor 2018.4.23f1.

**Comment by Xele02** [comment]
Looks like the game don't find those files which should be in the data zip :
F:\UMO\Tools\Data/db/ar_event!sa4130fcbz!.dat
F:/UMO/Tools/Data/db/md-20220626-000000_v00007790_s1_h00000000!s602bc890z!.dat 
F:/UMO/Tools/Data/db/sd!s6b10b98cz!.dat

**Comment by Akhmadi14** [comment]
Ok, seems like I got some error on the external hard-drive where the main data is stored (some files got corrupted including the one you mention above) which causing those errors.
I'm gonna sort the problems then check it out again.

**Comment by Xele02** [comment]
Any news ? I'll close the bug in 1 week considering it's fine now
